_pytest/test_components.py

```diff
@@ -63,5 +80,5 @@ def test_all_arguments_can_be_satisfied_during_parse(component_class, default_co
         for ctx_arg in clz.context_provides.get("process", []):
             context_arguments[ctx_arg] = None
 
-    filled_args = fill_args(component.process_args(), context_arguments, config.DEFAULT_CONFIG)
+    filled_args = fill_args(component.process_args(), context_arguments, default_config.as_dict)
```

Comment:
good catch :D fixed it. didn't throw an error because all parameters where satisfied from the context_arguments ;)